services/src/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/FlowHelper.groovy

```diff
@@ -261,6 +262,28 @@ class FlowHelper {
         }
     }
 
+    /**
+     * Check that flow state is changed correctly after corresponding flow history action.
+     *
+     * @param flowId
+     * @param timeFrom history will requested from this timestamp
+     * @param event sequence of flow history actions
+     * @param state expected flow state
+     */
+    void verifyFlowState(String flowId, Long timeFrom, FlowHistoryEvent event, state) {
+        def timeTo = timeFrom ? System.currentTimeSeconds() : null
```

Comment:
well, this is kind of tricky. timeTo based on timeFrom, why would you do this? Very strange and not intuitive approach